pycontw/pycon.tw
# Delete old travis setting

## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [X] **Other (Clear non required documents)**

## Description
Delete non required part of .travis.yml.

## Steps to Test This Pull Request
Steps to test new .travis.yml
1. Go to [https://travis-ci.com/](travis-ci.com)
2. Login personal Github account
2. Click on 'Dashboard'
3. Click on 'personal Github account/pycon.tw' rebuild button
3. See result is passed or errored

## Expected behavior
The result after Travis CI running is passed.

## Related Issue
[Official announcement of travis Travis CI]
Please be aware travis-ci.org will be shutting down in several weeks, with all accounts migrating to **travis-ci.com**. Please stay tuned here for more information.

![image](https://user-images.githubusercontent.com/20749936/113555250-3150b680-962d-11eb-8b96-c8a353d33078.png)


## More Information
**Test result**
![image](https://user-images.githubusercontent.com/20749936/113555364-68bf6300-962d-11eb-805d-e17d997c965b.png)
